base/harmful/harmful.c

```diff
@@ -260,35 +260,4 @@ TRAP(mq_timedreceive)
 TRAP(wordexp)
 TRAP(wordfree)
 
-/// C11 threading primitives are not supported by ThreadSanitizer.
-/// Also we should avoid using them for compatibility with old libc.
-TRAP(thrd_create)
```

Comment:
These changes are useful and should not be reverted

Reply:
the whole file or particular lines?..